.gitignore

```diff
@@ -216,3 +216,4 @@ mp/src/game/shared/neo/neo_version_info.cpp
 
 # News cache
 mp/game/neo/news.txt
+/mp/src/materialsystem/stdshaders/buildhl2mpshaders.bat
```

Comment:
Possibly a typo, since changes to `mp/src/materialsystem/stdshaders/buildhl2mpshaders.bat` are also included in this PR. Did you mean to ignore the path without the leading `/`?

Reply:
I changed the file before noticing it should be added to gitignore so I assumed the change would be added regardless, but maybe instead this is the reason why, lemme check

Reply:
I remember adding this file to the gitignore by right clicking it in visual studios Git Changes, which is why its placed so haphazardly and has the leading slash. I removed it and moved the entry to the other shader entries. I don't think the slash has any effect